composer.json

```diff
@@ -22,6 +22,13 @@
         {"name": "Jonathan Wage", "email": "jonwage@gmail.com"},
         {"name": "Marco Pivetta", "email": "ocramius@gmail.com"}
     ],
+    "support": {
+        "email": "doctrine-user@googlegroups.com",
+        "issues": "https://github.com/doctrine/doctrine2/issues",
+        "source": "https://github.com/doctrine/doctrine2",
```

Comment:
Here I hesitated with https://github.com/doctrine/doctrine2/tree/master/lib/Doctrine/ORM, and I notice that Packagist already displays a link pointing to https://github.com/doctrine/doctrine2/tree/master. Tell me what you think is best.

Reply:
Leave it like this, otherwise it'd be another maintenance burden.